runtime/compiler/optimizer/J9Inliner.hpp

```diff
@@ -239,12 +239,16 @@ class TR_J9InlinerPolicy : public OMR_InlinerPolicy
 class TR_J9JSR292InlinerPolicy : public TR_J9InlinerPolicy
    {
    friend class TR_J9InlinerUtil;
+   friend class TR_J9InlinerPolicy;
    friend class TR_InlinerBase;
    friend class TR_MultipleCallTargetInliner;
    public:
       TR_J9JSR292InlinerPolicy(TR::Compilation *comp);
    protected:
       virtual TR_InlinerFailureReason checkIfTargetInlineable(TR_CallTarget* target, TR_CallSite* callsite, TR::Compilation* comp);
+      static bool isJSR292Methods(TR_ResolvedMethod *resolvedMethod);
+      static bool isJSR292AlwaysInlineableMethods(TR_ResolvedMethod *resolvedMethod);
+      static bool isJSR292SmallGetterMethods(TR_ResolvedMethod *resolvedMethod);
```

Comment:
The pluralized methods does not make sense here given the parameter is only a single method. These APIs need to be documented.

Reply:
Changed query names as suggested and added documentation. thanks!